dev/tasks/linux-packages/apache-arrow/yum/arrow.spec.in

```diff
@@ -506,86 +506,6 @@ Libraries and header files for Gandiva.
 %{_libdir}/pkgconfig/gandiva.pc
 %endif
 
-%if %{use_python}
```

Comment:
There are still `%if %{use_python}` in this file. Could you also remove them?
Could you also remove `%define use_python (%{rhel} >= 8)`?